nip57.ts

```diff
@@ -68,9 +68,23 @@ export function makeZapRequest({
     ],
   }
 
-  if (event) {
+  if (event && typeof event === 'string') {
     zr.tags.push(['e', event])
   }
+  if (event && typeof event === 'object') {
+    zr.tags.push(['e', event.id])
+    // replacable event
+    if ((10000 <= event.kind  && event.kind < 20000) || event.kind  == 0 || event.kind  == 3){
```

Comment:
`isReplaceableKind` would be better.
https://github.com/nbd-wtf/nostr-tools/blob/1e5bfe856bfe953eb834e4bfa906cec5aed8cc88/kinds.ts#L9